ui/packages/atlasmap/src/UI/dialogs/ParametersDialog.stories.tsx

```diff
@@ -127,12 +127,26 @@ const parameters = [
   { name: 'quote', label: 'Quote', value: '', required: false, enabled: false },
 ];
 
+/**
+ * The 'Select' parameters dialog doesn't specify predefined parameters, the
+ * 'Edit' dialog does.
+ *
+ * @returns
+ */
 export const parametersDialog = () => (
-  <ParametersDialog
-    title={text('Title', 'Select CSV Processing Parameters')}
-    isOpen={boolean('Is open', true)}
-    onCancel={action('onCancel')}
-    onConfirm={action('onConfirm')}
-    parameters={parameters}
-  />
+  <span>
```

Comment:
Not sure why it's not showing both, but would be better to have separate story anyway, see https://github.com/atlasmap/atlasmap/blob/master/ui/packages/atlasmap/src/Views/MappingTableView.stories.tsx#L29-L45 for having multiple in a same file

Reply:
corrected